src/transformers/models/oneformer/modeling_oneformer.py

```diff
@@ -1556,7 +1556,7 @@ def __init__(self, config: OneFormerConfig):
             # for backwards compatibility
             backbone_config = MaskFormerSwinConfig.from_dict(backbone_config.to_dict())
             backbone_config.out_features = ["stage1", "stage2", "stage3", "stage4"]
```

Comment:
This doesn't need to be added, for OneFormer we'll directly rely on Swin

SwinBackbone is going to be added as part of this PR: https://github.com/huggingface/transformers/pull/20648

Reply:
Yes, I added these to make it work for now. I will change the code once DinatBackbone and SwinBackbone have been added. Please let me know when those PRs are merged.